### ISSUE TITLE
What's the correct way to use Monokai Pro with Markdown Editing?

### ISSUE THREAD

**Issue by dahanbn** [Monokai/monokai-pro-sublime-text#36]
Hi,

I am somewhat unsure how to correctly use Monokai Pro with Markdown Editing. Usually Markdown Editing has his own themes and I want to override them and use Monokai Pro.

Should I set syntax specific settings? What's the difference between both color schemes?

     "color_scheme": "Packages/User/Monokai Pro (Filter Octagon) (SublimePythonIDE).tmTheme",
     "color_scheme": "Packages/User/SublimeLinter/Monokai Pro (Filter Octagon) (SL).tmTheme",

Somehow I am stuck with a Monokai Pro Octagon variant even when I remove both of the mentioned settings. What would be the correct approach?

**Comment by dahanbn** [comment]
I just noticed that I can choose various Monokai Pro themes located under sublime linter, color highlighter, etc.

As it seems they also bevave differently. See the attached picture and the highlighting of bold.

![monokai_markdown](https://user-images.githubusercontent.com/4658596/30989179-14bfac18-a49d-11e7-94c8-00e1ca3b13ec.png)

**Comment by Monokai** [comment]
I do know that SublimeLinter converts themes on the fly, it appends `(SL)` to the file name. I don't know how Color Highlighter does this, as I've never used this plugin. I also see that you use something that appends `(SublimePythonIDE)` to the theme. Maybe you're using a plugin that also converts a theme on the fly.

The recommended way for Monokai Pro is to select the theme via the command palette: `command palette > Monokai Pro: select theme > Monokai Pro`. Sublime Linter handles its own converting afterwards. I can imagine other packages do the same.

**Comment by dahanbn** [comment]
I am still confused. If I change the theme via `command palette > Monokai Pro: select theme > Monokai Pro` than I will get a modified theme from Sublime Linter in my main settings file, e.g. `"color_scheme": "Packages/User/SublimeLinter/Monokai Pro (SL).tmTheme"`.

Somehow the Monokai theme switcher doesn't work in combination with Sublime Linter. 

Also if I use Markdown Editing than your switcher won't change the Markdown theme at all. Only via `UI: select colorscheme` I am able to change the colour scheme for Markdown. What would be the right name for your octagon color scheme that's not modified by any package?

**Comment by Monokai** [comment]
So what happens is this: you select a Monokai Pro theme via the command palette. Directly afterwards SublimeLinter processes the newly selected theme and appends `(SL)` to it. This doesn't matter, that's how SublimeLinter works (it adds some new rules to highlight linting errors). So now the theme setting is `Monokai Pro (SL)` (or any other filter variant with `SL` appended to it). This is fine.

So, now the Markdown theme. The standard Markdown syntax highlighting works. I think the complications arise from using the package MarkdownEditing.

I installed the package to test. Apparently it uses its own themes. What I would expect is that when I select the command `MarkdownEditing: Change color scheme` and select a color scheme, that it would be applied. I get an instant preview when I navigate the MarkdownEditing theme list (actually looks good with the option `Monokai Pro (SL)`), but the moment I select a theme, it reverts to its default theme `MarkdownEditor`.

I also tried to use the plugin with another SublimeText theme set, but I got the same results. So I have the feeling the MarkdownEditing plugin has a bug here. It doesn't seem like the themes can be selected?
